stylesheet.css

```diff
@@ -0,0 +1,257 @@
+body {
+  height: 100vh;
+  width: 100vw;
```

Comment:
Typically you don't need to apply sizing to the `body` directly. You should instead be setting sizing on the direct children of the body and the body's sizing is determined by the content. For example in your standard layout there is a header, main content, and footer. You get much better results if you set `height` on each of the individual pieces. For your project you could set `main-container` to `100vh` and achieve the same result.